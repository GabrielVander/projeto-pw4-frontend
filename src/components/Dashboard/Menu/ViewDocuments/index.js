import React from "react";
import Card from "react-bootstrap/Card";
import {BsFiles} from "react-icons/bs";
import { motion } from "framer-motion"
import {useHistory} from "react-router-dom";

function ViewDocuments() {
    const history = useHistory();

    return (
        <>
            <motion.div
                whileHover={{ scale: 1.1 }}
                whileTap={{ scale: 0.9 }}
            >
                <Card bg='warning' onClick={() => history.push('/documents')}>
                    <Card.Header>View documents</Card.Header>
                    <Card.Body>
                        <BsFiles size={50} />
                    </Card.Body>
                </Card>
            </motion.div>
        </>
    );
}

export default ViewDocuments;