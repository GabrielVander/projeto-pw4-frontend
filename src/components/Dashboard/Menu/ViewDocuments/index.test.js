import React from 'react';
import { render } from '@testing-library/react';
import ViewDocuments from './index';

describe('Component renders properly', () => {
	test('Matches snapshot', () => {
		const component = render(<ViewDocuments/>);

		expect(component).toBeDefined();
		expect(component).toMatchSnapshot();
	});
});