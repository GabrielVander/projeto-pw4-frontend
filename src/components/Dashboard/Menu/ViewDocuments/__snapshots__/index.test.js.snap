// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders properly Matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="card bg-warning"
        >
          <div
            class="card-header"
          >
            View documents
          </div>
          <div
            class="card-body"
          >
            <svg
              fill="currentColor"
              height="50"
              size="50"
              stroke="currentColor"
              stroke-width="0"
              viewBox="0 0 16 16"
              width="50"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                clip-rule="evenodd"
                d="M3 2h8a2 2 0 012 2v10a2 2 0 01-2 2H3a2 2 0 01-2-2V4a2 2 0 012-2zm0 1a1 1 0 00-1 1v10a1 1 0 001 1h8a1 1 0 001-1V4a1 1 0 00-1-1H3z"
                fill-rule="evenodd"
              />
              <path
                d="M5 0h8a2 2 0 012 2v10a2 2 0 01-2 2v-1a1 1 0 001-1V2a1 1 0 00-1-1H5a1 1 0 00-1 1H3a2 2 0 012-2z"
              />
            </svg>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="card bg-warning"
      >
        <div
          class="card-header"
        >
          View documents
        </div>
        <div
          class="card-body"
        >
          <svg
            fill="currentColor"
            height="50"
            size="50"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 16 16"
            width="50"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M3 2h8a2 2 0 012 2v10a2 2 0 01-2 2H3a2 2 0 01-2-2V4a2 2 0 012-2zm0 1a1 1 0 00-1 1v10a1 1 0 001 1h8a1 1 0 001-1V4a1 1 0 00-1-1H3z"
              fill-rule="evenodd"
            />
            <path
              d="M5 0h8a2 2 0 012 2v10a2 2 0 01-2 2v-1a1 1 0 001-1V2a1 1 0 00-1-1H5a1 1 0 00-1 1H3a2 2 0 012-2z"
            />
          </svg>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
